components/becomeSponsorButton.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div id="sponzaButtonContainer">
+    <img
+      src="~/assets/graphics/leafLeft.svg"
+    >
+    <a id="sponzaButton" href="mailto:logistics@nwplus.io?subject=CMD-F Sponsorship">
+      Become a Sponsor
+    </a>
+    <img
+      src="~/assets/graphics/leafRight.svg"
+    >
+  </div>
+</template>
+
+<style scoped lang="scss">
+@import "bulma/bulma.sass";
+#sponzahref {
+    width: 100%;
+    height: 100%;
+}
+
+#sponzoButtonWrapper {
+  display: inline-block;
+  margin: auto;
+}
+#sponzaButton {
+  text-decoration: none;
+  transition-duration: 0.3s;
```

Comment:
imo should be shorter

Reply:
@ianmah How about now?

Reply:
I was referring to the transition-duration
I haven't actually pulled your branch so I never even saw how short/long the button is haha

Reply:
just looked at it and the button looks a little bit small compared to figma

Reply:
can you make it look right? I suck so much at matching the figma lol

Reply:
aightttt